string project_name
string project_description
string owner